src/main/resources/settings.yaml

```diff
@@ -1,11 +1,12 @@
 port: ${FHIRSPARK_PORT:-3001}
-fhirDbBase: ${FHIRSPARK_FHIRBASE:-http://localhost:8080/fhir}
+fhirDbBase: ${FHIRSPARK_FHIRBASE:-http://localhost:8080/fhir/}
 specimenSystem: ${FHIRSPARK_SPECIMENSYSTEM:-https://cbioportal.org/specimen/}
 diagnosticReportSystem: ${FHIRSPARK_DIAGNOSTICREPORTSYSTEM:-https://cbioportal.org/mtb/}
 observationSystem: ${FHIRSPARK_OBSERVATIONSYSTEM:-https://cbioportal.org/therapyrecommendation/}
+followUpSystem: ${FHIRSPARK_FOLLOWUPSYSTEM:-https://cbioportal.org/followup/}
 patientSystem: ${FHIRSPARK_PATIENTSYSTEM:-https://cbioportal.org/patient/}
 serviceRequestSystem: ${FHIRSPARK_SERVICEREQUESTSYSTEM:-https://cbioportal.org/servicerequest/}
-portalUrl: ${FHIRSPARK_PORTALURL:-http://localhost:8080/}
+portalUrl: ${FHIRSPARK_PORTALURL:-http://localhost:8080/fhir/}
```

Comment:
Why is this changed? This URL should point to cBioPortal and it looks like it is not being used in your codebase anyway.